lib/generators/curation_concerns/install_generator.rb

```diff
@@ -101,8 +101,11 @@ def resource_types_config
       copy_file "config/authorities/resource_types.yml", "config/authorities/resource_types.yml"
     end
 
-    def simple_form_bootstrap
-      generate 'simple_form:install --bootstrap'
```

Comment:
I just added this a few weeks ago. It's not something sufia 6 did. So clearly we don't need it.